### DISCUSSION TITLE
Latest PCB schematics

### DISCUSSION THREAD

**Discussion by kachurovskiy** [kachurovskiy/nanoels#22] [Show and tell]
For the folks who want to make the circuit board themselves: ![Schematic_NanoEls_2021-12-04](https://user-images.githubusercontent.com/517919/145374884-ec55aafa-20af-4558-b182-93eb9b7c119c.png)

**Comment by ws786873** [comment]
Is there any difference between this schematic diagram and the previous one?

**Reply by kachurovskiy** [reply]
I've removed unused pins (second stepper connection and analog in) and simplified everything as much as possible. This schematics is posted as a reply to a YouTube comment asking for one :)

**Reply by kachurovskiy** [reply]
Also, supplying 5V via the 5V pin and not Vin - apparently Vin should be used with higher voltages (starting with 7V).

**Reply by ws786873** [reply]
OK, thank you for your explanation. I have ordered your original PCB。
